PixiEditor/Models/Controllers/BitmapManager.cs

```diff
@@ -43,9 +43,16 @@ public Layer PreviewLayer
 
         public Color PrimaryColor { get; set; }
 
-        public int ToolSize => SelectedTool.Toolbar.GetSetting("ToolSize") != null
-            ? (int) SelectedTool.Toolbar.GetSetting("ToolSize").Value
+        public int ToolSize
+        {
+            get => SelectedTool.Toolbar.GetSetting("ToolSize") != null
+            ? (int)SelectedTool.Toolbar.GetSetting("ToolSize").Value
             : 1;
+            set
+            {
+                SelectedTool.Toolbar.GetSetting("ToolSize").Value = value;
```

Comment:
Using shortcuts on the tool, which doesn't have `ToolSize` raises an error, you should add null checking on the setter.